stories/topics/proc-gcp-update-remove-data-file.adoc

```diff
@@ -0,0 +1,14 @@
+[id="proc-gcp-update-remove-data-file"]
```

Comment:
Would it make sense to change the ID and filename, so that the URL for the published doc agrees with them?